ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -20,17 +20,20 @@ use ykaddr::addr::symbol_vaddr;
 // This is simple and can be shared across both IRs.
 pub(crate) use super::aot_ir::Predicate;
 
-/// A fixed-width integer type.
+/// A fixed-width integer.
```

Comment:
I think this was right before? We are talking about the type of a fixed width integer, not an instance of a fixed-width integer.

Reply:
Agreed. Fixed in 9052c1b6.